base/linalg/uniformscaling.jl

```diff
@@ -138,6 +138,7 @@ function (-)(J::UniformScaling{TJ}, A::AbstractMatrix{TA}) where {TA,TJ<:Number}
 end
 
 inv(J::UniformScaling) = UniformScaling(inv(J.λ))
+norm(J::UniformScaling, p::Real=2) = abs(J.λ)
```

Comment:
This should include a check that `p` is 1, 2, or Inf similar to the check in `linalg/generic.jl`.